README.md

```diff
@@ -7,20 +7,182 @@ This module consists of information about the artist and track as well as the ab
 
 ## Related Projects
 
-- https://github.com/4-ever-young/soundclout-active-player-module.git
-- https://github.com/4-ever-young/soundclout-active-player-module.git
-- https://github.com/4-ever-young/Sidebar-module.git
+- https://github.com/AirCloudy/song-display
+- https://github.com/AirCloudy/playbar
+- https://github.com/AirCloudy/Related-Tracks
 
 ## Table of Contents
 
 1. [Usage](#Usage)
 1. [Requirements](#requirements)
+1. [API Call](#apicall)
 1. [Development](#development)
 
 ## Usage
 
 The user will be able to view track and artist information, post comments, like and repost tracks and comments.
 
+## API Call
+
+
+### POST
+
+#### POST /song
+Add a new song under the song collection.
+JSON object goes in the body.
+
+**Body**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| 'song_id' | 'int' | _Required_. Identifier for song. |
```

Comment:
just an fyi, if you're looking for code formatting use backticks instead of quotation marks